### PR TITLE
Translation of 2024-04-23 published news to ko

### PR DESCRIPTION
**Pull request by riseshia**
:link: https://github.com/ruby/www.ruby-lang.org/issues/2818

Translation of 2024-04-23 published news to ko.

- [CVE-2024-27282: Arbitrary memory address read vulnerability with Regex search](https://github.com/ruby/www.ruby-lang.org/blob/master/en/news/_posts/2024-04-23-arbitrary-memory-address-read-regexp-cve-2024-27282.md)
- [Ruby 3.0.7 Released](https://github.com/ruby/www.ruby-lang.org/blob/master/en/news/_posts/2024-04-23-ruby-3-0-7-released.md)
- [Ruby 3.1.5 Released](https://github.com/ruby/www.ruby-lang.org/blob/master/en/news/_posts/2024-04-23-ruby-3-1-5-released.md)
- [Ruby 3.2.4 Released](https://github.com/ruby/www.ruby-lang.org/blob/master/en/news/_posts/2024-04-23-ruby-3-2-4-released.md)
- [Ruby 3.3.1 Released](https://github.com/ruby/www.ruby-lang.org/blob/master/en/news/_posts/2024-04-23-ruby-3-3-1-released.md)
- #3226

Translation diff is [f0f97d1](https://github.com/ruby/www.ruby-lang.org/pull/3228/commits/f0f97d13b641981f3e9da16abf29f4216c487817)

### CODE REVIEW

File: ko/news/_posts/2024-04-23-arbitrary-memory-address-read-regexp-cve-2024-27282.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+tags: security
+lang: ko
+---
+
+정규표현식 검색의 임의의 메모리 주소 읽기 취약점에 대한 보안 수정을 포함하는 Ruby 3.0.7, 3.1.5, 3.2.4, 3.3.1을 릴리스했습니다.
+이 취약점은 CVE 번호 [CVE-2024-27282](https://www.cve.org/CVERecord?id=CVE-2024-27282)로 등록되어 있습니다.
+
+## 세부 내용
+
+Ruby 3.0.0부터 3.3.0까지 문제가 발견되었습니다.
```
Comment (yous) [inline ko/news/_posts/2024-04-23-arbitrary-memory-address-read-regexp-cve-2024-27282.md:16]: Maybe we can just use `3.x` as is?

```suggestion
Ruby 3.x부터 3.3.0까지 문제가 발견되었습니다.
```

File: ko/news/_posts/2024-04-23-ruby-3-0-7-released.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+Ruby 3.0.7이 릴리스되었습니다.
+
+이번 릴리스에는 보안 수정 사항이 포함되어 있습니다.
+자세한 내용은 아래 항목을 참조하세요.
+
+* [CVE-2024-27282: 정규표현식 검색의 임의의 메모리 주소 읽기 취약점]({%link ko/news/_posts/2024-04-23-arbitrary-memory-address-read-regexp-cve-2024-27282.md %})
+* [CVE-2024-27281: RDoc에서 .rdoc_options 사용 시의 RCE 취약점](https://www.ruby-lang.org/ko/news/2024/03/21/rce-rdoc-cve-2024-27281/)
+* [CVE-2024-27280: StringIO에서 버퍼 초과 읽기 취약점](https://www.ruby-lang.org/ko/news/2024/03/21/buffer-overread-cve-2024-27280/)
+
+자세한 내용은 [GitHub 릴리스](https://github.com/ruby/ruby/releases/tag/v3_0_7)을 참조하세요.
```
Comment (yous) [inline ko/news/_posts/2024-04-23-ruby-3-0-7-released.md:19]:
```suggestion
자세한 내용은 [GitHub 릴리스](https://github.com/ruby/ruby/releases/tag/v3_0_7)를 참조하세요.
```

File: ko/news/_posts/2024-04-23-ruby-3-0-7-released.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+이 릴리스가 끝나면 Ruby 3.0은 EOL에 도달합니다. 즉, 이번 릴리스가 Ruby 3.0 버전대의 마지막 릴리스가 될 것입니다.
```
Comment (yous) [inline ko/news/_posts/2024-04-23-ruby-3-0-7-released.md:21]:
```suggestion
이 릴리스 후에 Ruby 3.0은 EOL에 도달합니다. 즉, 이번 릴리스가 Ruby 3.0 버전대의 마지막 릴리스가 될 것입니다.
```

File: ko/news/_posts/2024-04-23-ruby-3-0-7-released.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+보안 취약점이 발견되더라도 Ruby 3.0.8는 릴리스되지 않을 것입니다. (심각한 회귀 버그가 발생하는 경우는 예외입니다.)
```
Comment (yous) [inline ko/news/_posts/2024-04-23-ruby-3-0-7-released.md:22]:
```suggestion
보안 취약점이 발견되더라도 Ruby 3.0.8은 릴리스되지 않을 것입니다. (심각한 회귀 버그가 발생하는 경우는 예외입니다.)
```

File: ko/news/_posts/2024-04-23-ruby-3-0-7-released.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+---
+
+Ruby 3.0.7이 릴리스되었습니다.
+
+이번 릴리스에는 보안 수정 사항이 포함되어 있습니다.
+자세한 내용은 아래 항목을 참조하세요.
+
+* [CVE-2024-27282: 정규표현식 검색의 임의의 메모리 주소 읽기 취약점]({%link ko/news/_posts/2024-04-23-arbitrary-memory-address-read-regexp-cve-2024-27282.md %})
+* [CVE-2024-27281: RDoc에서 .rdoc_options 사용 시의 RCE 취약점](https://www.ruby-lang.org/ko/news/2024/03/21/rce-rdoc-cve-2024-27281/)
+* [CVE-2024-27280: StringIO에서 버퍼 초과 읽기 취약점](https://www.ruby-lang.org/ko/news/2024/03/21/buffer-overread-cve-2024-27280/)
```
Comment (yous) [inline ko/news/_posts/2024-04-23-ruby-3-0-7-released.md:17]: These links can be `{%link ... %}` too, so `en` version also can get fixed. It's same with 3.1.5, 3.2.4, and 3.3.1 posts too.

File: ko/news/_posts/2024-04-23-arbitrary-memory-address-read-regexp-cve-2024-27282.md (hunk 1 above)
Comment (riseshia) [reply ko/news/_posts/2024-04-23-arbitrary-memory-address-read-regexp-cve-2024-27282.md]: 👍

File: ko/news/_posts/2024-04-23-ruby-3-0-7-released.md (hunk 5 above)
Comment (riseshia) [reply ko/news/_posts/2024-04-23-ruby-3-0-7-released.md]: fixed 1e9d46c7044405b5e79d1475a0ea6b15189d9ec6

**Review by yous — APPROVED** [review]

File: ko/news/_posts/2024-04-23-ruby-3-0-7-released.md (hunk 5 above)
Comment (riseshia) [reply ko/news/_posts/2024-04-23-ruby-3-0-7-released.md]: and also en \w https://github.com/ruby/www.ruby-lang.org/pull/3231

**Comment by riseshia** [comment]
Thanks~

**Pull request merged**